torch/csrc/jit/passes/subgraph_matcher.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
+
+#include <torch/csrc/jit/ir.h>
+
+#include <unordered_map>
+#include <unordered_set>
```

Comment:
You also seem to be using `<vector>` and `<iostream>` (which are probably transitively included from `ir.h`, but it's better to include them explicitly)?